tests/integration/stepfunctions/state_machine_templates.py

```diff
@@ -0,0 +1,58 @@
+# TODO: Temporary collection of state machine templates. Options:
+#   - Enable dumping of Component objects for machine definitions.
+#   - Create sub-templates for state machine sub productions.
+from localstack.utils.json import clone
+
+
+class Templates:
+    BASE_INVALID_DER = {
+        "Comment": "BASE_INVALID_DER",
+        "StartAt": "State_1",
+    }
+
+    BASE_PASS_RESULT = {
+        "Comment": "BASE_PASS_RESULT",
+        "StartAt": "State_1",
+        "States": {
+            "State_1": {
+                "Type": "Pass",
+                "Result": {
+                    "Arg1": "argument1",
+                },
+                "End": True,
+            }
+        },
+    }
```

Comment:
Just summarizing again what we discussed yesterday:

We have more or less 3 approaches here:
1. Python (as-is)
2. JSON files
3. JSON5 files

In terms of preference I'd say it would be 3 => 1 => 2, since adding comments to the samples is quite valuable. Unfortunately this means we'll need to transform them back into normal JSON first before passing them to a create_statemachine call, but would be quite nice to see a small prototype for this :)

(Not high priority right now and **this is not blocking the merge!**)